sonic_platform_base/sonic_xcvr/xcvr_eeprom.py

```diff
@@ -91,3 +104,24 @@ def write_raw(self, offset, size, bytearray_data):
          Boolean, True if the write is successful and False otherwise
       """
       return self.writer(offset, size, bytearray_data)
+   
+   def refresh_cache(self, **kwargs):
+      """
+      When invoked, it caches the lower and upper pages of 00h i.e offset 0 to 256
+      Any read request made strictly b/w these limits will be read from the cache until cleared
+
+      Returns: 
+         None
+      """
+      # TODO: Use a better mechanism and allow caching for higher pages and more complicated scenarios
```

Comment:
I think we should consider to have  solution which is field specific rather than offset or page specific. The 'if' checks here will increase if we have static and non-static fields in same page.